addons/event/views/event_registration_views.xml

```diff
@@ -108,36 +108,44 @@
                 <templates>
                     <t t-name="event_attendees_kanban_icons_desktop">
                         <div class="d-none d-md-block h-100">
-                            <div id="event_attendees_kanban_icons_desktop" class="h-100 float-end p-2 d-flex align-items-end flex-column">
+                            <div id="event_attendees_kanban_icons_desktop" class="h-100 float-end p-2 d-flex align-items-end flex-column gap-1">
                                 <t t-if="record.active.raw_value">
-                                    <a class="btn btn-md btn-primary" string="Confirm Registration" name="action_confirm" type="object" states="draft" role="button">
+                                    <a class="btn btn-md btn-secondary" string="Confirm Registration" name="action_confirm" type="object" states="draft" role="button">
                                         <i class="fa fa-check" role="img" aria-label="Confirm button" title="Confirm Registration"/>
                                     </a>
                                     <a class="btn btn-md btn-primary" string="Confirm Attendance" name="action_set_done" type="object" states="open" role="button">
-                                        <i class="fa fa-user-plus" role="img" aria-label="Attended button" title="Confirm Attendance"/>
+                                        <i class="fa fa-check" role="img" aria-label="Attended button" title="Confirm Attendance"/>
                                     </a>
                                     <span class="text-muted" states="done">Attended</span>
                                     <span class="text-muted" states="cancel">Canceled</span>
                                 </t>
+                                <a class="btn btn-md btn-secondary" string="Previous state" name="action_previous_state" type="object" role="button"
```

Comment:
Now that I think about it, this view changes imply changes for translation no ? Can you check and update the `pot` file where it's needed if so 😄